Packs/MongoDB/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### MongoDB
+
+- Updated the ***mongodb-update*** command to support the *upsert* argument, which allows to insert a new entry in case no entry matched the query.
```

Comment:
```suggestion
- Updated the ***mongodb-update*** command to support the *upsert* argument, which allows you to insert a new entry in case no entry matched the query.
```

Reply:
Fixed.